### ISSUE TITLE
ThemeDictionary is defined as ThemeDictionnary

### ISSUE THREAD

**Issue by daxpandhi** [Simnico99/MicaWPF#62]
The current example code failed to work, then I found out `mica:ThemeDictionary` was actually `ThemeDictionnary`. This is v5.2.1.

**Comment by Simnico99** [comment]
??? That is how I use it:

```xaml
<Application
    x:Class="TaskForceUtilities.WPF.Launcher.App"
    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
    xmlns:mica="clr-namespace:MicaWPF.Styles;assembly=MicaWPF">
    <Application.Resources>
        <ResourceDictionary>
            <ResourceDictionary.MergedDictionaries>
                <mica:ThemeDictionnary Theme="Auto" />
                <mica:ControlsDictionnary />
            </ResourceDictionary.MergedDictionaries>
        </ResourceDictionary>
    </Application.Resources>
</Application>
```

Can you provide a more precise explanation or code snipet?

**Comment by daxpandhi** [comment]
Yes. But `ThemeDictionnary` has 2 n, where it should have 1. Once fixed it is fine. 

I saw that this was referenced in https://github.com/Simnico99/MicaWPF/issues/55#issuecomment-1463708039 - but the typo still seems to be there in 5.2.1.

**Comment by daxpandhi** [comment]
![image](https://user-images.githubusercontent.com/6795809/225915804-9c05496a-255b-4154-8fc5-f9732602f5ac.png)
![image](https://user-images.githubusercontent.com/6795809/225915847-250138bb-91ce-49ed-a76d-99166e2e33bc.png)


Super minor thing, but just wanted to mention because readme says "Dictionary" so anyone pasting the code may get confused. :)

**Comment by Simnico99** [comment]
Pushing V5.2.2 It will take time but I removed one N seems that the merged request didn't merge properly last time.

**Comment by Simnico99** [comment]
https://github.com/Simnico99/MicaWPF/releases/tag/5.2.2

**Comment by daxpandhi** [comment]
Great! Thank you so much!

May I ask you one more question. Don't want to make a new bug if it is my fault.

When compiling for .NET 7, I get this error: 

> MicaWPF.dll cannot be referenced because it uses built-in support for WinRT, which is no longer supported in .NET 5 and higher.  An updated version of the component supporting .NET 5 is needed. For more information, see https://aka.ms/netsdk1149

**Comment by Simnico99** [comment]
Modify your Project.cs file to have these 2 lines
<TargetFramework>net7.0-windows10.0.19041.0</TargetFramework>
<TargetPlatformMinVersion>7.0</TargetPlatformMinVersion>

Because it uses WinRT to communicate with an assembly you need the -windows10.0.19041.0 at the end

**Comment by daxpandhi** [comment]
Thanks!
